reports/default/json-schema.json

```diff
@@ -58,6 +58,17 @@
             "minLength": 4,
             "maxLength": 128
         },
+        "currency_type": {
+            "type": "string",
+            "pattern": "[A-Z]+",
+            "minLength": 3,
+            "maxLength": 3,
+            "enum": [
+                "RUB",
+                "EUR",
+                "USD"
```

Comment:
`null`?

Reply:
nullable defenition? I think `null` should not be defined here